protostar/commands/test/testing_summary.py

```diff
@@ -61,6 +61,24 @@ def log(
                 + str(self.testing_seed.value)
             )
 
+    def log_slowest(
```

Comment:
```suggestion
    def log_slowest_test_case(
```
Public methods should be more explicit. Somebody could think that this method logs THE slowest test case.